src/sentry/event_manager.py

```diff
@@ -888,6 +883,13 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
         else:
             is_sample = can_sample
 
+        if not is_sample:
+            GroupHash.record_last_processed_event_id(
```

Comment:
I considered try/excepting this but hadn't made up my mind yet whether that's a good idea.